### PR TITLE
Add draft layout of the app

### PR DESCRIPTION
**Pull request by Diogomartf**
This PR is meant to draft the main layout of the app.

**Landing**
image

**My bets**
image

**Question page**
image

### CODE REVIEW

File: next.config.mjs [hunk 1]
```diff
@@ -1,4 +1,6 @@
 /** @type {import('next').NextConfig} */
-const nextConfig = {};
+const nextConfig = {
+  distDir: "dist",
+};
```
Comment (berteotti) [inline next.config.mjs:4]: Let's add this too, so we can export the app to IPFS and fix webpack lib issues.

```suggestion
const nextConfig = {
  distDir: "dist",
  output: "export",
  webpack(config) {
    config.resolve.fallback = { "pino-pretty": false, encoding: false };

    return config;
  },
};
```

File: app/my/page.tsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import { RowCard } from "@/app/components/ui";
+import Link from "next/link";
+
+export default function MyPage() {
```
Comment (berteotti) [inline app/my/page.tsx:4]: "My" is a bit vague, my-bets would be better. Other option is to call it portfolio.

Comment (Diogomartf) [reply app/my/page.tsx:4]: ok, we can change it to my bets

Comment (berteotti) [reply app/my/page.tsx:4]: We will show only bets there, so I think my-bets works fine for now. Or are you going to show more things under `/my/`?

Comment (Diogomartf) [reply app/my/page.tsx:4]: just bets

**Review by berteotti — APPROVED** [review]

**Pull request merged**